src/PHPSemVerChecker/Console/Application.php

```diff
@@ -25,7 +25,7 @@ public function __construct()
 		parent::__construct('PHP Semantic Versioning Checker by Tom Rochette', self::VERSION);
 
 		// Suppress deprecated warnings
-		error_reporting(E_ALL ^E_DEPRECATED);
+		error_reporting(E_ALL & ~E_DEPRECATED);
```

Comment:
Hi @tomzx 
Please note that while this suppress E_DEPRECATED in "standalone" application  there are still deprecation messages generated if this module is used as dependency. Would be great if this PR could be merged and new version released so that issue resolved everywhere

Reply:
Hello, I've removed the error silencing in v0.15.0. Let me know if this fixed your issue.